TemplateApplicationTests/TemplateApplicationTests.swift

```diff
@@ -6,12 +6,33 @@
 // SPDX-License-Identifier: MIT
 //
 
+import SwiftUI
 @testable import TemplateApplication
 import XCTest
 
 
 class TemplateApplicationTests: XCTestCase {
-    func testExample() throws {
-        XCTAssertTrue(true)
+    func testNegateBinding() throws {
```

Comment:
The negate binding is now part of SpeziViews. So one could also remove that piece of code from the template application.

Reply:
Nice; thank you! Will remove this 🚀